opsimulator/opsimulator.go

```diff
@@ -411,6 +414,20 @@ func (opSim *OpSimulator) checkInteropInvariants(ctx context.Context, logs []typ
 			if common.BytesToHash(executingMessage.MsgHash[:]).Cmp(initiatingMsgPayloadHash) != 0 {
 				return fmt.Errorf("executing and initiating message fields are not equal")
 			}
+
+			if opSim.interopDelay != 0 {
+				// Add time check after getting the initiating message block header
+				executingBlockHeader, err := opSim.ethClient.HeaderByNumber(ctx, nil)
```

Comment:
s/executingBlockHeader/header

Lets make the variable names a bit less verbose, the executingBlock doesn't really mean anything here. We're just checking again the latest head of the chain